Makefile

```diff
@@ -43,6 +44,7 @@ NODE_EXE = node$(EXEEXT)
 NODE ?= ./$(NODE_EXE)
 NODE_G_EXE = node_g$(EXEEXT)
 NPM ?= ./deps/npm/bin/npm-cli.js
+TEST_CI = ./tools/test-ci.sh
```

Comment:
> @refack git is not a requirement for testing, tho

so a dual runner, `git tool` and `node tool`. One for bootstrapping, one for post-build tooling.
Could even throw in a `python tool` for the extreme minimalists...